COPYRIGHT.txt

```diff
@@ -130,6 +130,11 @@ Comment: Basis Universal
 Copyright: 2019, Binomial LLC.
 License: Apache-2.0
 
+Files: ./thirdparty/brotli/
+Comment: Brotli
+Copyright: 2009, 2010, 2013-2016 by the Brotli Authors.
+License: MIT
```

Comment:
```suggestion
License: Expat
```

That's the name used by Debian in their spec, as MIT is often ambiguous (or _was_, nowadays I think everyone means the same with MIT, but well): https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-short-name